src/app/api/audit-log/message/route.ts

```diff
@@ -0,0 +1,54 @@
+
+import {NextResponse, NextRequest} from 'next/server';
+
+
+interface FetchedMessage {
+    id: string;
+    from: string;
+    to: string;
+    direction: string;
+    timestamp: string;
+}
+
+export async function GET(
+    request: NextRequest,
+    res: NextResponse
+) {
+   const accountSid = process.env.TWILIO_ACCOUNT_SID;
+   const authToken = process.env.TWILIO_AUTH_TOKEN;
+   const client = require('twilio')(accountSid, authToken);
+   try{
+    console.log('success')
+    const messages: any[] = await client.messages.list({limit:20});
+    //if want to display all calls, can delete the limit parameter
+    const formattedMessages: FetchedMessage[] = messages.map(message => {
+        const date = new Date(message.dateCreated);
```

Comment:
same comments as above - removing console logs and using dayjs for date formatting